src/common/lockdep.cc

```diff
@@ -68,7 +72,7 @@ void lockdep_register_ceph_context(CephContext *cct)
     if (!free_ids_inited) {
       free_ids_inited = true;
       // FIPS zeroization audit 20191115: this memset is not security related.
```

Comment:
probably want to remove the FIPS note

Reply:
`free_ids.set()` is practically identical to `memset()`. i don't think we should drop this comment.

Reply:
I got the impression that the comment was related to an automated search for the text 'memset' that
was intended to identify memsets in specific scenarios. SO probably not relevant to any other clearing
mechanisms.

In any case - a very minor issue. Your call.

Reply:
@rzarzynski what do you think? is it safe to drop it and another comment?

Reply:
@ronen-fr removed and repushed.